modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -836,9 +884,20 @@ export class Tileset3D {
     this.stats.get(TILES_LOADED).incrementCount();
     this.stats.get(TILES_IN_MEMORY).incrementCount();
 
-    // Good enough? Just use the raw binary ArrayBuffer's byte length.
+    // TODO: Calculate GPU memory usage statistics for a tile.
     this.gpuMemoryUsageInBytes += tile.gpuMemoryUsageInBytes || 0;
+
     this.stats.get(TILES_GPU_MEMORY).count = this.gpuMemoryUsageInBytes;
+
+    // Adjust SSE based on cache limits
+    if (this.options.memoryAdjustedScreenSpaceError) {
+      if (this.gpuMemoryUsageInBytes < this._cacheBytes) {
+        this.decreaseScreenSpaceError();
```

Comment:
If this is the only please increase and decrease is made, I am not sure having those methods defined separately and far away is worth it. Maybe just break this entire function out into `adjustScreenSpaceError()`